htdocs/comm/propal/class/propal.class.php

```diff
@@ -4443,8 +4443,17 @@ public function insert($notrigger = 0)
 				// End call triggers
 			}
 
-			$this->db->commit();
-			return 1;
+			if (!$error) {
+				$this->db->commit();
+				return $this->id;
```

Comment:
Here the return value was modified.
Before you got "return 1" and then you get "return $this->id" and it can be null

Reply:
haa... you've spotted it.
In all other 'Line" class the insert method already return the rowid of the new line created, so it should be the same here, but I underantd that it is a change not a fix. I revert it and made another PR for develop